Bit of an update to the directions

Drauku opened the pull request:
I reordered the guide directions to improve readability and fixed some formatting inconsistencies.
My overall intention was to make the guide easier to follow.

If I did this PR incorrectly, just send it back to me and I'll try to figure out what I did wrong!

gkoerk committed:
3 New Recipes: BitWarden, Bookstack, and Dozzle

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
traefik changes

gkoerk committed:
Elaborate on removal of CS and Container shared folder.

gkoerk committed:
Cleanup old recipes which have been migrated to a new repo

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
Update README.md

gkoerk committed:
added librepay option

Drauku committed:
Bit of an update to the Guide, tried to improve readability and ordering of the steps.

Drauku committed:
Fixed some 'shortcut' script formatting discrepancies

Drauku committed:
separator above section 4 was in wrong spot

Drauku committed:
note at end of section 3 now correctly indented

Drauku committed:
tweaked section delimiters

Drauku committed:
re-added traefik acme.json updated permissions instructions

Drauku committed:
Format and Auth0 update

Major format update, as well as several Traefik and Auth0 section tweaks.
THERE ARE STILL ISSUES WITH Auth0, and possibly Traefik config file names.

Drauku committed:
Readme scripts update

MAJOR scripts update, minor readability tweaks and script reference updates.

Drauku committed:
Merge branch 'master' of https://github.com/Drauku/QNAP-Docker-Swarm-Setup

Drauku committed:
swarm scripts upload

Drauku committed:
added .env symlink command to dsd

Drauku committed:
scripts fix for 'Segmentation Fault' error

Drauku committed:
variables list update

Drauku committed:
directions order change, to include cmd shortcuts

Drauku committed:
fixed traefik single bounce action in scripts

Drauku committed:
original yml upload NOT written with variables

Drauku committed:
converted *most* env variables to use common vars

Drauku committed:
fixed 'swarm_vars' file extension

Drauku committed:
comment update for elasticsearch 'sysctl' command

Drauku committed:
Revert "fixed 'swarm_vars' file extension"

This reverts commit 2faecd162ec8a0fd15cbe78330a8cea2d15d45f0.

Drauku committed:
fixed swarm_vars.conf internal reference var name

Drauku committed:
split script_vars apart from swarm_vars

Drauku committed:
updated var source to script_vars from swarm_vars

Drauku committed:
re-added nas variable info

gkoerk commented on README.md:
```diff
@@ -92,49 +83,12 @@ Please consider donating above to support the [QNAP Unofficial Discord](https://
+2. Install nano or vi, whichever you are more comfortable with (only one needed)
     - **RUN:** `opkg install nano`
     - **RUN:** `opkg install vim`
     - ***NOTE:*** You must have installed the `entware-std` package as detailed above in Section-2 Step-8 to be able to use the "opkg" installer.
```
Where are the instructions to install the `entware-std` package?

gkoerk commented on README.md:
```diff
@@ -343,26 +341,33 @@ These variable/config files need to be filled in with your information in order
 ## Each listed stack will require a corresponding '/stackname/stackname.yml' folder/file in the 'configs' folder defined below
 ## NOTE: Leave Traefik off the list as it will be started seperately
 stacks_default=(
-  graylog
```
Why remove graylog?

gkoerk commented on README.md:
```diff
@@ -343,26 +341,33 @@ These variable/config files need to be filled in with your information in order
+  bitwarden
   ddclient
   docker-cleanup
-  ouroboros
```
Why remove ouroboros?

gkoerk commented on README.md:
```diff
@@ -343,26 +341,33 @@ These variable/config files need to be filled in with your information in order
 # List desired services inside the 'stacks' array parentheses (each service name separated by at least a space)
 ## Each listed stack will require a corresponding '/stackname/stackname.yml' folder/file in the 'configs' folder defined below
 ## NOTE: Leave Traefik off the list as it will be started seperately
 stacks_default=(
```
Shouldn't traefik be in the list of defaults?

gkoerk reviewed (commented):
I need more information to discuss these changes. A lot seems specific to your setup.

Drauku committed:
readibility and flow updates

Drauku committed:
added 'dwinit' profile command

dwinit creates basic swarm folder structure

Drauku committed:
Merge branch 'master' of https://github.com/Drauku/QNAP-Docker-Swarm-Setup

Drauku committed:
scripts and variable files update

Drauku committed:
add docker_commands_list script

Drauku committed:
remove / at end of folder config vars

Drauku committed:
delete blank/extra readme.md file

Drauku committed:
mass removal of all ${} variables

Drauku committed:
updated docker_stack_folders.sh alias filename

Drauku committed:
removed old docker_configs.sh

Drauku committed:
fixed /share/swarm/config typo

Drauku committed:
Readme format overhaul

Drauku committed:
missed a paragraph

Drauku committed:
Create docker_compose_logs.sh

This script displays 50 log entries for the indicated docker-compose container.

Drauku committed:
add docker_compose_logs shortcut reference
